Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2103,7 +2103,7 @@ def gen2018HiMix(fragment,howMuch):
                                       '--geometry' : geom
                                       }
 
-    upgradeStepDict['RecoFullGlobal'][k] = {'-s':'RAW2DIGI,L1Reco,RECO,PAT,VALIDATION:@phase2Validation+@miniAODValidation,DQM:@phase2+@miniAODDQM',
+    upgradeStepDict['RecoFullGlobal'][k] = {'-s':'RAW2DIGI,L1Reco,RECO,RECOSIM,PAT,VALIDATION:@phase2Validation+@miniAODValidation,DQM:@phase2+@miniAODDQM',
```

Comment:
2017 classic mixing is generated from RecoFull step (above) please add RECOSIM there

Reply:
this update [for "RecoFull"] is still missing.

Reply:
Fixed. Tested on 10042.0,10842.0,11642.0.